nautobot/docs/release-notes/version-1.2.md

```diff
@@ -120,6 +120,12 @@ The Admin sub-site within Nautobot (`/admin/` and its child pages) has been reva
 
 Job log messages are now stored in a separate database table as a separate `JobLogEntry` data model, instead of being stored as JSON on the `JobResult` model/table. This provides faster and more robust rendering of `JobResult`-related views and lays groundwork for future enhancements of the Jobs feature.
 
+!!! note
+    If you use Jobs inside tests, your Tests Class should have `@mock.patch("nautobot.extras.models.models.JOB_LOGS", None)`. This will allow the tests and the JobLogEntry objects to use the `default` database.
+
+!!! note
+    If you are using the antipattern `job.job_result = JobResult()` in tests, this will no longer work. Instead you will need to create the JobResult object `job.job_result = JobResult.objects.create(...)`.
```

Comment:
```suggestion
    Because `JobLogEntry` records reference their associated `JobResult`, the pattern `job.job_result = JobResult()` (creating only an in-memory `JobResult` object, rather than a database entry) will no longer work. Instead you will need to create a proper JobResult database object `job.job_result = JobResult.objects.create(...)`.
```